Networking/NetworkingTests/Mapper/OrderMapperTests.swift

```diff
@@ -197,6 +197,23 @@ final class OrderMapperTests: XCTestCase {
         XCTAssertEqual(partialRefund2.total, "-8.10")
     }
 
+    /// Verifies that an Order ignores deleted refunds.
+    ///
+    func test_Order_deleted_refund_fields_are_ignored() {
```

Comment:
nit: would be nice to add the [Given/When/Then](https://en.wikipedia.org/wiki/Given-When-Then) comments (or a subset of them) in the test case, similar to the test case below (`test_taxes_are_parsed_correctly`) 🙂

Reply:
Sounds good. Added in https://github.com/woocommerce/woocommerce-ios/pull/3349/commits/055eb6e19065116d2ae4f4eed2c8428a62d26ba5
Thank you!